integration-tests/.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Integration Tests",
+            "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
```

Comment:
This launch config doesn't support the other part of running tests in here, which is `dtslint` (see scripts in `package.json`). Could we support the larger `npm run test` script, or is that kind of wild and not helpful for debugging?

Reply:
launch.json doesn't support `npm run test` unfortunately. I did just add a new config named `Integration Type Checker`. This runs `dtslint`. 

Sidenote: When I run `dtslint` in my CLI, I'm not seeing any output except blank console.logs. I'm seeing the same when I run the configuration in vscode.